tests/ui-api-layer-acceptance-tests/installer/broker.go

```diff
@@ -0,0 +1,132 @@
+package installer
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"fmt"
+	"log"
+	"time"
+
+	"github.com/kubernetes-incubator/service-catalog/pkg/apis/servicecatalog/v1beta1"
+	"github.com/kubernetes-incubator/service-catalog/pkg/client/clientset_generated/clientset"
+	"github.com/kyma-project/kyma/tests/ui-api-layer-acceptance-tests"
+	"github.com/kyma-project/kyma/tests/ui-api-layer-acceptance-tests/waiter"
+)
+
+const (
+	brokerReadyTimeout = time.Second * 300
+)
+
+type BrokerInstaller struct {
+	name      string
+	namespace string
+	typeOf    string
+}
+
+func NewBroker(name, namespace, typeOf string) *BrokerInstaller {
+	return &BrokerInstaller{
+		name:      name,
+		namespace: namespace,
+		typeOf:    typeOf,
+	}
+}
+
+func (t *BrokerInstaller) Install(svcatCli *clientset.Clientset, releaseName string) error {
+	url := "http://" + releaseName + "." + t.namespace + ".svc.cluster.local"
+
+	var err error
+	if t.typeOf == tester.ClusterServiceBroker {
+		_, err = svcatCli.ServicecatalogV1beta1().ClusterServiceBrokers().Create(upsClusterServiceBroker(t.name, url))
```

Comment:
If in every exported method we have `if / else`, then it would be better to split the structs to `ServiceBrokerInstaller` and `ClusterServiceBrokerInstaller`

Reply:
Also, I don't like the fact that the UPS broker is hardcoded in the installer. Please pass the `ServiceBroker` and `ClusterServiceBroker` from the top. `BrokerInstaller` shouldn't have `upsClusterServiceBroker` and `upsServiceBroker` methods.

Reply:
Done...